examples/llm/src/models/mosaic_gpt.py

```diff
@@ -99,6 +99,15 @@ def __init__(self, cfg: DictConfig):
         else:
             self.attn_mask = None
 
+        if cfg.get('no_bias', False):
+            for module in self.modules():
+                if isinstance(module, (nn.Linear, nn.LayerNorm)):
+                    if module.bias is not None:
+                        if isinstance(module.bias, nn.Parameter):
+                            print(f'Removing bias {module.bias}', end=' ')
+                            module.register_parameter('bias', None)
+                            print(f'from {module}.')
```

Comment:
+100 style points

Reply:
As much as I like this, should we gate it somehow with like `verbose: true`? Otherwise anytime we init the model there could be like 100 lines of "Removing bias..."

Reply:
Thank you

Reply:
We don't have a verbose flag being passed to model.
Should I create one or nuke the print?

Reply:
I'll add a `verbose` flag; it'll potentially be useful in other places.

Reply:
adding `verbose` flag. It might be useful later as well.